lib/prometheus/que_stats.rb

```diff
@@ -101,7 +101,7 @@ def expired
       JOB_CLASSES = %w[ApplicationJob ProcessEntryJob ProcessIntegrationEntryJob UpdateJob].inspect.tr('"', "'").freeze
       private_constant :JOB_CLASSES
 
-      JOB_CLASS_COLUMN_NAME = 'job'
+      JOB_CLASS_COLUMN_NAME = 'job_name'
```

Comment:
`job` is a reserved label.
It was not validated before this update

https://github.com/yabeda-rb/yabeda-prometheus/compare/v0.1.4...v0.1.5#diff-22016af103dc675d6642ae3cc94786811e87be9e35d53e38fbc416a011168fd8R27